.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @btrautmann @ClaireDavis @Betterment/test_track_core
```

Comment:
@btrautmann i'm adding you and claire explicitly, and then the group for fallback.

since we're switching to policy-bot, i updated the PR template to no longer have the domain/platform callouts.